### PR TITLE
Clarify concept exercise naming rules

### PR DESCRIPTION
**Pull request by SaschaMann**
See https://github.com/exercism/docs/pull/72/files#r585601262

### CODE REVIEW

File: anatomy/tracks/concept-exercises.md [hunk 1]
```diff
@@ -473,19 +473,19 @@ Each exercise also has a _slug_, which is normalized version of the exercise nam
 
 1. Use lowercase.
 1. Use [kebab-case][kebab-case].
-1. Use alphanumerical characters ('a' - 'z') and dashes ('-').
+1. Use latin alphabetical characters ('a' - 'z') and dashes ('-').
```
Comment (iHiD) [inline anatomy/tracks/concept-exercises.md]:
```suggestion
1. Use latin alphanumerical characters and dashes (Regexp: `[a-z0-9-]+`)
1. Prefer written digits over numeric ones, unless there is a specific reason to prefer the digit (e.g. `two-fer` over `2-fer`)
```

**Review by ErikSchierboom — APPROVED** [review]
Great! Could you do the same for the exercise names (not the slug) directly above this section?

**Comment by SaschaMann** [comment]
Done

**Review by iHiD — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Brilliant. Thanks!

**Pull request merged**